superset-frontend/src/views/CRUD/data/dataset/AddDataset/index.tsx

```diff
@@ -96,9 +97,10 @@ export default function AddDataset() {
       .then(json => {
         setDatasets(json?.result);
       })
-      .catch(error =>
-        logging.error('There was an error fetching dataset', error),
-      );
+      .catch(error => {
+        addDangerToast('There was an error fetching dataset');
```

Comment:
```suggestion
        addDangerToast(t('There was an error fetching dataset'));
```

Reply:
Fixed in [`this commit`](https://github.com/apache/superset/pull/22610/commits/a72d05829371f6a375e165dd2d3739474587fa63).